psi4/src/psi4/libmints/potentialint.cc

```diff
@@ -27,12 +27,22 @@
  */
 
 #include "psi4/libmints/potentialint.h"
+#include "psi4/libpsi4util/process.h"
+#include "libint2/engine.h"
 
 namespace psi {
 
 PCMPotentialInt::PCMPotentialInt(std::vector<SphericalTransform>& trans, std::shared_ptr<BasisSet> bs1,
                                  std::shared_ptr<BasisSet> /* bs2 */, int /* deriv */)
     : PotentialInt(trans, bs1, bs1) {
+
+    int max_am = std::max(basis1()->max_am(), basis2()->max_am());
```

Comment:
Any special reason this integral needs to re-define from the original `PotentialInt`?

Reply:
It uses some functor magic to handle the two types of contraction needed in PCM.  It's likely possible to use the original object, but this is a pretty lightweight class.